src/app/assets/icons/ClassBallDoubleDigit.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+type ClassBallProps = {
+  num: string;
```

Comment:
`num` is a `string`?
This doesn't sound right. 
I recommend to not use abbreviations here.
Wdyt about:

```suggestion
  digit: number;
```

Reply:
It is in fact a string, since it is the description of the courselevel. I agree that this naming is a bit confusing, so I changed it to course. Thank you.